gradle/libs.versions.toml

```
@@ -6,6 +6,7 @@ okio = "3.0.0-alpha.10"
 serialization = "1.3.1"
 kotlinx-datetime = "0.3.1"
 kotlinx-coroutines = "1.5.2"
+kotlin-zip4j = "2.9.1"
```

```suggestion
zip4j = "2.9.1"
```
This is pure Java library? Why prefix it with `kotlin-`?

---

Fixed